2021/ml4co-competition/index.html

```diff
@@ -641,11 +633,11 @@ <h2>Metrics</h2>
 				<h2>Datasets</h2>
 			</header>
 			<p>
-				For each individual challenge, the participants will be
+				The participants will be
 				evaluated on <b>three problem benchmarks</b> from diverse application
-				areas. Participants can provide a different
+				areas for each challenge. Participants can provide a different
```

Comment:
I like the old sentence better. It is a bit confusing that we have at the same time three tasks and three benchmarks. That's why I tried to be very explicit here: for each individual challenge, the three benchmarks will be used.